build.gradle.kts

```diff
@@ -75,17 +75,20 @@ buildscript {
 plugins {
     `jvm-module`
     protobuf
+    ksp
+    `module-testing`
 }
 
 // Cannot use `id()` syntax for McJava because it's not yet published to the Plugin Portal
-// and is added to the build classpath via `buildScript` block above.
+// and is added to the build classpath via the `buildScript` block above.
 apply(plugin = "io.spine.mc-java")
 
 apply<IncrementGuard>()
 apply<VersionWriter>()
 
+apply<BomsPlugin>()
+
 apply(from = "$projectDir/version.gradle.kts")
```

Comment:
They can be grouped: 

```kotlin
apply { 
    plugin("io.spine.mc-java)
    plugin<IncrementGuard>()
    plugin<VersionWriter>()
    plugin<BomsPlugin>()
    from("$projectDir/version.gradle.kts")
}
```

Reply:
It can, but I'd rather not to. The order of applying the plugins and their semantics are a bit different. Also, `plugin { id("blah") }` and `apply { plugin("blah") }` work differently regarding DLS appearing in the build script.

With Gradle, it's usually "don't touch when it works". So, let's move with new things gradually.